core/src/edu/asu/c3simulator/util/PlayerInterfaceImplementation.java

```diff
@@ -0,0 +1,89 @@
+package edu.asu.c3simulator.util;
+
+import edu.asu.c3simulator.simulation.C3Simulation;
+import edu.asu.c3simulator.simulation.Player;
+import edu.asu.c3simulator.simulation.SocioeconomicStatus;
+
+/**
+ * Implementation of the Player interface Provides detail about various player
+ * attributes that are needed for PlayerStatusDisplay
+ * 
+ * @author Reigel, Justin
+ *
+ */
+
+public class PlayerInterfaceImplementation implements Player
+{
+	private SocioeconomicStatus status;
+	private int netWorth;
+	private int capital;
+	private C3Simulation simulation;
+
+	public PlayerInterfaceImplementation(SocioeconomicStatus newStatus, int newNetWorth,
+			int newCapital, C3Simulation newSimulation)
+	{
+		status = newStatus;
+		netWorth = newNetWorth;
+		capital = newCapital;
+		simulation = newSimulation;
+	}
```

Comment:
Honestly, I can delete it if needed and make a new one in sprint 4. It is not used anywhere. It was used in MainHub, but then I never got around to answering any of those questions and removed it from MainHub.